<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>200</width>
      <height>34</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Detector&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="det_name"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>591</width>
      <height>256</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Minimum Integrated Counts of a Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Maximum Integrated Counts of a Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Saturation Count of a Pixel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Maximum Number of Saturated Pixels of a Frame</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Saturation Count Rate of a Pixel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Maximum Number of Count Rate Saturation Pixels of a Frame</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="sum_ll"/>
       </item>
       <item>
        <widget class="QLineEdit" name="sum_hl"/>
       </item>
       <item>
        <widget class="QLineEdit" name="pix_sat_hl"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Npix_sat_hl"/>
       </item>
       <item>
        <widget class="QLineEdit" name="pix_sat_cnt_rate_hl"/>
       </item>
       <item>
        <widget class="QLineEdit" name="Npix_sat_cnt_rate_hl"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="list_failed">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>591</width>
      <height>121</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_list">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>490</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVerifier">
    <property name="title">
     <string>Verifier</string>
    </property>
    <addaction name="actionStart_verifier"/>
    <addaction name="actionStop_verifier"/>
   </widget>
   <addaction name="menuVerifier"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart_verifier">
   <property name="text">
    <string>start verifier</string>
   </property>
  </action>
  <action name="actionStop_verifier">
   <property name="text">
    <string>stop verifier</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
